js/ui/camera.js

```diff
@@ -475,10 +475,6 @@ util.extend(Camera.prototype, /** @lends Map.prototype */{
             this.pitching = true;
         }
 
-        if (this.zooming && !around) {
-            around = tr.pointLocation(tr.centerPoint.add(to.sub(from).div(1 - 1 / scale)));
-        }
```

Comment:
As far as I can tell, this code is not needed. There may also be a solution that involves incorporating bearing into the above calculation.